microbootstrap/settings/base.py

```diff
@@ -3,28 +3,18 @@
 
 import pydantic
 import pydantic_settings
-from sentry_sdk.integrations import Integration  # noqa: TCH002
 
-from microbootstrap.helpers.logging import base as logging_base
-from microbootstrap.helpers.opentelemetry import OpenTelemetryInstrumentor  # noqa: TCH001
+from microbootstrap.base.logging import base as logging_base
+from microbootstrap.instruments import OpentelemetryInstrumentConfig, SentryInstrumentConfig
 
 
-class BootstrapSettings(pydantic_settings.BaseSettings):
+class BootstrapSettings(pydantic_settings.BaseSettings, SentryInstrumentConfig, OpentelemetryInstrumentConfig):
     debug: bool = False
     app_environment: str | None = None
-    namespace: str = "default"
-    service_name: str = pydantic.Field(default="service")
-    service_version: str = pydantic.Field(default="0")
+    service_name: str = pydantic.Field(default="micro-service")
+    service_version: str = pydantic.Field(default="1.0.0")
```

Comment:
Version was 0 for a reason: it screamed “change me!” 😁

Reply:
Maybe, remove the default?

Reply:
I think the main idea of ​​this library is this: you don't have to do almost anything to make the service work.
It seems that they will be able to change the value themselves to something more valid if they really need it
And version 1.0.0 indicates that the service has just been created and there have been no updates yet, which is true at the very beginning, as I believe

Reply:
I think that our task in this lib is to provide such default values ​​so that they do not hurt the eyes, but everything works with them